ui/admin/client/scripts/pages/execution-metrics.js

```diff
@@ -71,70 +71,135 @@ var Controller = [
+          )}&endDate=${encodeURIComponent(fixDate($scope.endDate))}`,
         {
-          flowNodes: function(cb) {
-            MetricsResource.sum(
-              {
-                name: 'activity-instance-start',
-                startDate: fixDate($scope.startDate),
-                endDate: fixDate($scope.endDate)
-              },
-              function(err, res) {
-                cb(err, !err ? res.result : null);
-              }
-            );
-          },
-          decisionElements: function(cb) {
-            MetricsResource.sum(
-              {
-                name: 'executed-decision-elements',
-                startDate: fixDate($scope.startDate),
-                endDate: fixDate($scope.endDate)
-              },
-              function(err, res) {
-                cb(err, !err ? res.result : null);
-              }
-            );
-          },
-          rootProcessInstances: function(cb) {
-            MetricsResource.sum(
-              {
-                name: 'root-process-instance-start',
-                startDate: fixDate($scope.startDate),
-                endDate: fixDate($scope.endDate)
-              },
-              function(err, res) {
-                cb(err, !err ? res.result : null);
-              }
-            );
-          },
-          decisionInstances: function(cb) {
-            MetricsResource.sum(
-              {
-                name: 'executed-decision-instances',
-                startDate: fixDate($scope.startDate),
-                endDate: fixDate($scope.endDate)
+          headers: {
+            method: 'GET',
+            'Content-Type': 'application/json'
+          }
+        }
+      ).then(
+        function(res) {
+          if (res.status >= 200 && res.status < 300) {
+            res.json().then(
+              function(res) {
+                cb(null, res.result);
               },
-              function(err, res) {
-                cb(err, !err ? res.result : null);
+              function(err) {
+                cb(err, null);
               }
             );
+          } else {
+            cb(res, null);
           }
         },
-        function(err, res) {
-          $scope.loadingState = 'LOADED';
-          if (err) {
-            setLoadingError('Could not set start and end dates.');
+        function(err) {
+          cb(err, null);
+        }
+      );
+    }
+
+    $scope.$watch('isTaskWorkerMetric', function() {
+      if ($scope.isTaskWorkerMetric) {
+        $scope.loadingState = 'LOADING';
+        fetchTaskWorkerMetric(function(err, result) {
+          if (!err) {
+            $scope.loadingState = 'LOADED';
+            $scope.metrics.taskWorkers = result;
+          } else {
+            setLoadingError('Could not load task worker metrics.');
             $scope.loadingState = 'ERROR';
-            return;
           }
-          $scope.metrics = res;
+          updateView();
+        });
+      } else {
+        $scope.loadingState = 'LOADED';
```

Comment:
This causes the loadingState to be `LOADED` on inital load. Therefore, no Loading indicator is displayed. 
We could change this either by setting the loading state only if `isTaskWorkerMetric` is not `undefined` or by replacing the Checkbox with a button which is hidden after the loading.

![localhost_8080_camunda_app_admin_default_ (1)](https://user-images.githubusercontent.com/21984219/94696446-550e0980-0337-11eb-8543-b1eb636acc88.png)